libs/gi/sheets/src/Weapons/Sword/SturdyBone/index.tsx

```diff
@@ -19,14 +19,16 @@ const normal_dmgInc = equal(
   )
 )
 
+const dmg = equal(input.weapon.key, key, normal_dmgInc)
```

Comment:
You can just wrap the equal above with this statement in normaldmginc, no need to make another var